%YAML:1.2
---
# model_type: PINHOLE
# camera_name: camera
# image_width: 1226
# image_height: 370
PlanMap:
  resolution: 0.1 # (unit: m)
  sogm_size_x: 15 # (unit: m)
  sogm_size_y: 15 # (unit: m)
  sogm_size_z: 5 # (unit: m)
  p_hit: 0.80
  p_miss: 0.35
  p_min: 0.12
  p_max: 0.97
  p_occ: 0.80

  inflate_size_xy: 2 #(unit: resolution)
  inflate_size_z: 1 #(unit: resolution)

DepthCamera:
  heigth: 640
  width: 480
  fx: 392.365234375
  fy: 392.365234375
  cx: 320.01055908203125
  cy: 241.12890625

  k_depth_scaling_factor: 1000
  depth_maxdist: 5.0 # (unit: m)
  depth_mindist: 0.1 # (unit: m)
  depth_filter_margin: 10
  skip_pixel: 4

  R_C_2_B: !!opencv-matrix #normal realsense
    rows: 3
    cols: 3
    dt: d
    data: [0, 0, 1, -1, 0, 0, 0, -1, 0]

  T_C_2_B: !!opencv-matrix
    rows: 1
    cols: 3
    dt: d
    data: [0.0, 0.0, 0.0]
